worker-test/src/test/java/com/github/workerframework/workertest/PoisonMessageIT.java

```diff
@@ -69,6 +68,40 @@ public void getWorkerNameInPoisonMessageTest() throws Exception {
                     .build();
 
             channel.basicPublish("", WORKER_IN, properties, codec.serialise(requestTaskMessage));
+            
+            // Test worker gets killed, the message SHOULD get redelivered, but it is not
+            // so we will consume and nack the message, forcing a redelivery which
+            // will be handled as a poison message.
+            nackMessage(channel);
```

Comment:
If test worker gets killed, the message SHOULD get redelivered, but it is not so we will consume and nack the message, forcing a redelivery which will be handled as a poison message.